### PR TITLE
Prepare for new release

### PR DESCRIPTION
**Pull request by kozlovic**
Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,10 +565,27 @@ authorization {
 
 The NATS server supports authorization using subject-level permissions on a per-user basis. Permission-based authorization is available with [multi-user authentication](#authentication). See also the [Server Authorization](http://nats.io/documentation/server/gnatsd-authorization) documentation.
 
-Each permission grant is an object with two fields: what subject(s) the authenticated user can publish to, and what subject(s) the authenticated user can subscribe to. The parser is generous at understanding what the intent is, so both arrays and singletons are processed. Subjects themselves can contain wildcards. Permissions make use of [variables](#variables).
+Before server `1.3.0`, it was only possible to define permissions allowing an authenticated user to pubslih or subscribe to certain subjects. Starting with `1.3.0`, it is now possible to also define permissions denying the right to publish or subscribe to specific subjects.
```
Comment (derekcollison) [inline README.md]: Spelling - publish

Comment (derekcollison) [inline README.md]: Would point out wildcards and the fact that you can add as many as you like to allow or deny clauses.

File: README.md [hunk 2]
```diff
@@ -565,10 +565,27 @@ authorization {
...
+
+Each permission grant is an object with two fields: what subject(s) the authenticated user is allowed (or denied the right) to publish to, and what subject(s) the authenticated user is allowed (or denied the right) to subscribe to. The parser is generous at understanding what the intent is, so both arrays and singletons are processed. Subjects themselves can contain wildcards. Permissions make use of [variables](#variables).
```
Comment (derekcollison) [inline README.md]: The old style is still accepted as well, and these fields are not required.

File: README.md [hunk 3]
```diff
@@ -615,9 +631,34 @@ Alice is a REQUESTOR and can publish requests on subjects "req.foo" or "req.bar"
 
 Charlie has no permissions granted and therefore inherits the default permission set. You set the inherited default permissions by assigning them to the default_permissions entry inside of the authorization configuration block.
 
-Bob is a RESPONDER to any of Alice's requests, so Bob needs to be able to subscribe to the request subjects and respond to Alice's reply subject which will be an _INBOX.>.
+Bob is a RESPONDER to any of Alice's requests, so Bob needs to be able to subscribe to the request subjects and respond to Alice's reply subject which will be an "_INBOX.>".
+
+Important to note, in order to not break request/reply patterns you need to add rules as above with Alice and Bob for the "_INBOX.>" pattern. If an unauthorized client publishes or attempts to subscribe to a subject that is not in the allow-list, or is in the deny-list, the action fails and is logged at the server, and an error message is returned to the client.
+
+Most of the time it is fine to specify the subjects that an user is allowed to publish or subscribe to.
```
Comment (derekcollison) [inline README.md]: a user I think va an user.

File: README.md [hunk 4]
```diff
@@ -615,9 +631,34 @@ Alice is a REQUESTOR and can publish requests on subjects "req.foo" or "req.bar"
...
+However, in some instances, it is much easier to configure the subjects that an user is not allowed to publish/subscribe .
```
Comment (derekcollison) [inline README.md]: a user

Comment (derekcollison) [reply README.md]: Also no "allow" clause means everything.

File: README.md (hunk 1 above)
Comment (kozlovic) [reply README.md]: Is this comment valid since just below we talk about what a permission grant is?

File: README.md (hunk 2 above)
Comment (kozlovic) [reply README.md]: Same, I do mention that old style is supported below that.

**Pull request merged**